src/Cimpress.Auth0.Client/Proxies/RoAuthenticationRequestDto.cs

```diff
@@ -8,14 +8,14 @@ namespace Cimpress.Auth0.Client.Proxies
     /// <remarks>
     /// This has been copied and slightly adapterd from https://github.com/auth0/auth0.net since it doesn't yet support netcore.
     /// </remarks>
-    public class AuthenticationRequestDto
+    public class RoAuthenticationRequestDto
```

Comment:
I'm curious the reason for this name changed, does the `Ro` signal something different, but I'm not sure what.  The other ones are called `Token` and `RefrestTokenDelegation`.  More of question that review comment.